README.md

```diff
@@ -299,14 +310,16 @@ No local configuration is set
 
 More information about mbed CLI configuration is available in the [configuration section](#mbed-cli-configuration) of this document.
 
-#### Through mbed_settings.py
+#### Setting environment variable
 
-Edit `mbed_settings.py` to set your toolchain:
+For each of the compilers, there is a corresponding environment variable that is checked for the path to that compiler. The environment variables are as follows:
```

Comment:
What checks the environment variable for the path to that compiler? The compiler?